tests/test_run_reference_vmm.py

```diff
@@ -101,7 +100,8 @@ def start_vmm_process(kernel_path, disk_path=None, num_vcpus=1, mem_size_mib=102
         "--kernel", "cmdline={},path={},himem_start={}".format(
             cmdline, kernel_path, himem_start
         ),
-        "--vcpu", "num={}".format(num_vcpus)
+        "--vcpu", "num={}".format(num_vcpus),
+        "--serial", "serial_input={},serial_output={}".format(serial_in, serial_out)
```

Comment:
I think we shouldn't forward the `serial_input` and `serial_output` to the vmm process if they're the empty string. This will help with keeping the default behavior as we had it before.